src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -27,32 +27,21 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
   options.reconnectionHandler = options.reconnectionHandler || window['Blazor'].defaultReconnectionHandler;
   logger.log(LogLevel.Information, 'Starting up blazor server-side application.');
 
-  // Initialize statefully prerendered circuits and their components
-  // Note: This will all be removed soon
   const initialConnection = await initializeConnection(options, logger);
-  const circuits = discoverPrerenderedCircuits(document);
-  for (let i = 0; i < circuits.length; i++) {
-    const circuit = circuits[i];
-    for (let j = 0; j < circuit.components.length; j++) {
-      const component = circuit.components[j];
-      component.initialize();
-    }
-  }
-
   const circuit = await startCircuit(initialConnection);
   if (!circuit) {
-    logger.log(LogLevel.Information, 'No preregistered components to render.');
+    throw new Error('Ciruit failed to start.');
```

Comment:
I think this should be an error now, if you can't start a circuit that means something is wrong. Interested in feedback in case I'm wrong, but I think I thought about all of the cases.

Reply:
In that case, it would be clearer if `startCircuit` was not capable of returning a promise that can resolve as `null`, but instead was required to resolve with a value or an exception.

Currently it's declared as returning `Promise<CircuitDescriptor | undefined>`. Can we change that to `Promise<CircuitDescriptor>` and then throw from inside it if it doesn't have a value to resolve with?

Reply:
Also, typo in `Ciruit`.

Reply:
Sure.